.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+permissions: read-all
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  check-scripts:
+    name: Check scripts
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./scripts
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@main
+      - name: Install the latest version of uv
+        uses: eifinger/setup-uv@v1
+        with:
+          enable-cache: true
```

Since we run with `--frozen`, maybe we can also customize a bit the [cache invalidation](https://docs.astral.sh/uv/guides/integration/github/#caching)? 
```suggestion
          enable-cache: true
          cache-dependency-glob: "uv.lock"
```

---

Great suggestion!